src/GraphQL/Validation/INodeVisitor.cs

```diff
@@ -2,9 +2,19 @@
 
 namespace GraphQL.Validation
 {
+    /// <summary>
+    /// An interface to handle events raised by a node walker such as <see cref="BasicVisitor"/>.
```

Comment:
I'm not sure about _events_ wording.

Reply:
Events, while not technically C# "events", are easy to understand and portray what's going on rather well.  If you want to change it, that's fine, however.

Reply:
```suggestion
    /// An interface to handle methods called by a node walker such as <see cref="BasicVisitor"/>.
```